lib/asar/fs-wrapper.ts

```diff
@@ -646,7 +646,7 @@ export const wrapFsWithAsar = (fs: Record<string, any>) => {
       return;
     }
 
-    if (options.withFileTypes) {
+    if (options?.withFileTypes) {
```

Comment:
This line fails to compile due to unexpected token, maybe caused by old typescript version?

Reply:
yeah - we didn't backport the webpack update to 21 so @dsanders11 think you'll need to extract that bit

Reply:
I've updated this backport to not use optional chaining on this line, should fix the issue.